Graviton.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,5 +1,50 @@
 {
   "pins" : [
+    {
+      "identity" : "cwlcatchexception",
```

Comment:
I don't think I use it. It might have come from one of the new dependencies, maybe LASwift given https://github.com/AlexanderTar/LASwift/blob/master/LASwift.xcodeproj/GeneratedModuleMap/CwlCatchExceptionSupport/module.modulemap? I'm not an expert in Swift or its package management so let me know if there's a good way to debug this.

Reply:
Makes sense. It's fine if it is transitive dependency.